e2e/test_operator/operator_test.go

```diff
@@ -142,6 +144,79 @@ var _ = Describe("Operator", Label("e2e"), func() {
 		})
 	})
 
+	When("taint a Node that hosts a Pod", func() {
+		var replacedPod corev1.Pod
+		taintKeyMaintenance := "maintenance"
+		taintKeyMaintenanceDuration := int64(2)
+		taintKeyStar := "*"
+		taintKeyStarDuration := int64(5)
+
+		BeforeEach(func() {
+			initialPods := fdbCluster.GetStatelessPods()
+			replacedPod = fixtures.RandomPickOnePod(initialPods.Items)
+			replacedPodProcessGroupID := internal.GetProcessGroupIDFromMeta(fdbCluster.GetCluster(), replacedPod.ObjectMeta)
+			// Taint replacePod's node
+			// Q: Why do I have to GetNode before UpdateNode()? otherwise, I got invalid node.spec.id error
```

Comment:
In general it's a good idea to fetch the latest status of an object before doing any modifications.